src/main/resources/default/taglib/k/toc.html.pasta

```diff
@@ -0,0 +1,12 @@
+<i:arg type="String" name="headingKey" default="KnowledgeBaseArticle.toc_heading" description="May contain a i18n key." />
```

Comment:
Nur noch eine Frage :)

Gibts für `KnowledgeBaseArticle.toc_heading` schon Übersetzungen? Wenn nicht wären diese zumindest in Deutsch und Englisch wichtig.

Reply:
Habs. Füge es hinzu..